optimum/exporters/onnx/model_patcher.py

```diff
@@ -1438,3 +1457,37 @@ def __exit__(self, exc_type, exc_value, traceback):
             from transformers.models.cohere.modeling_cohere import CohereRotaryEmbedding
 
             CohereRotaryEmbedding.forward = self.original_forward
+
+
+def gpt_oss_forward(self, hidden_states: torch.Tensor, router_indices=None, routing_weights=None) -> torch.Tensor:
```

Comment:
would you mind adding a link to the original code like https://github.com/huggingface/transformers/blob/v4.56.0/src/transformers/models/gpt_oss/modeling_gpt_oss.py#L81